Docs/ProximityLights.md

```diff
@@ -74,42 +73,38 @@ In the following steps we will create a new material that is illuminated by a pr
 
 ## Advanced usage
 
-By default only three proximity lights can illuminate a material at a time. If your project requires an additional proximity light to influence a material the sample code below demonstrates how to achieve this.
+By default only three proximity lights can illuminate a material at a time. If your project requires additional proximity lights to influence a material the steps below demonstrates how to achieve this.
 
 > [!NOTE]
 > Having many proximity lights illuminate a material will increase pixel shader instructions and will impact performance. Please profile these changes within your project.
 
-*How to increase the number of available proximity lights from three to four.*
+*How to increase the number of proximity lights which effect a surface from three to six.*
```

Comment:
This line may work better with header formatting, to make it clear that all that follows is related.